### ISSUE TITLE
Output soft lesion segmentation

### ISSUE THREAD

**Issue by valosekj** [ivadomed/model_seg_sci#86]
SCIseg currently outputs binary lesion segmentation. This might not be always ideal. For example, the tissue bridges are usually computed on soft lesion segmentations (manually created using JIM). @jcohenadad thus suggested extending SCIseg to output the soft segmentation using dropout and Monte Carlo. After some investigation with @naga-karthik, this could be done by commenting on the [this line](https://github.com/MIC-DKFZ/nnUNet/blob/75c46fe7c1c85c60e020d76e387772c7be02a1af/nnunetv2/inference/predict_from_raw_data.py#L617C9-L617C28) in the nnunet source code and modifying the plans.json file:

```console
"dropout_op": torch.nn.Dropout,
"dropout _op_kwargs": 0.5,
```
